.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: '11:00'
+  open-pull-requests-limit: 10
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
You may want to switch this to weekly or monthly so you don't have to deal with a constant stream of python updates.

Reply:
There's no python code in the repo so I think this doesn't really matter

Reply:
I would just remove that section in that case.

I've added an update check for GitHub actions to the CE dependabot file; there are some examples of the config in the dependabot docs here: 

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
Fixed